### PR TITLE
cleanup finagle-postgres workdir after jar build

### PR DESCRIPTION
**Pull request by dt**
shaves almost 100mb of jars off virtual size

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -27,7 +27,7 @@ RUN npm install pg
 
 RUN curl -SL https://jdbc.postgresql.org/download/postgresql-9.4.1207.jar > /postgres.jar
 
-RUN git clone https://github.com/cockroachdb/finagle-postgres.git && cd finagle-postgres && git checkout fatjartests && ./sbt assembly && mv target/scala-2.11/finagle-postgres-tests.jar /
+RUN git clone https://github.com/cockroachdb/finagle-postgres.git && cd /finagle-postgres && git checkout fatjartests && ./sbt assembly && mv target/scala-2.11/finagle-postgres-tests.jar / && cd / && rm -rf /finagle-postgres
```
Comment (tamird) [inline Dockerfile]: how about some line breaks?

File: Dockerfile [hunk 2]
```diff
@@ -27,7 +27,12 @@ RUN npm install pg
 
 RUN curl -SL https://jdbc.postgresql.org/download/postgresql-9.4.1207.jar > /postgres.jar
 
-RUN git clone https://github.com/cockroachdb/finagle-postgres.git && cd finagle-postgres && git checkout fatjartests && ./sbt assembly && mv target/scala-2.11/finagle-postgres-tests.jar /
+RUN git clone https://github.com/cockroachdb/finagle-postgres.git && \
```
Comment (madelynnblue) [inline Dockerfile]: Could also add `--depth 1` here to reduce checkout time.

**Comment by madelynnblue** [comment]
LGTM

File: Dockerfile (hunk 2 above)
Comment (tamird) [inline Dockerfile]: +1. Also you can add `-b fatjartests` so you can skip the `git checkout` below

**Pull request merged**